src/App.tsx

```diff
@@ -15,48 +16,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+interface Sort {
```

Comment:
I think enum will be more suitable instead of interface

*Use enum SortType instead of string literals for possible sorting options (including the default one)*

Reply:
agree with [comment](https://github.com/mate-academy/react_list-of-goods-hooks/pull/1062/files#r1250579509). try to use `enum`